### ISSUE TITLE
Node.js 10.0.0 Tracking

### ISSUE THREAD

**Issue by jasnell** [nodejs/Release#291]
FYI... I will be kicking off the process for Node.js 10.0.0 the first week of January. Current plan is to create the actual `v10.x` and `v10.x-staging` branches on March 1st.

**Comment by mhdawson** [comment]
+1

**Comment by jasnell** [comment]
Ping @nodejs/release @nodejs/lts @nodejs/tsc .@nodejs/collaborators.. Because Node.js 10.x will be an LTS branch, it needs an LTS code name. The 8.x code name was "Carbon", which means we're on the D's. Unfortunately, the options for `D` are quite limited.

* `Darmstadtium` - "the synthetic chemical element of atomic number 110. It is a superheavy metal that decays in thousandths of a second."
* `Dubnium` - "the chemical element of atomic number 105, a very unstable element made by high-energy atomic collisions."
* `Dysprosium` - "the chemical element of atomic number 66, a soft silvery-white metal of the lanthanide series."

Aside from the horrible names, it would be exceedingly funny to name a Long Term Support line after either an element that decays in thousandths of a second, or after a highly unstable element, so either of the first too automatically get my vote.

Please weigh in on your choice.

**Comment by ofrobots** [comment]
My vote is none of the above. Let's pick another theme.

**Comment by jasnell** [comment]
> My vote is none of the above. Let's pick another theme.

aww ;-)

**Comment by kapouer** [comment]
??? Diamond !!!

**Comment by trivikr** [comment]
[Deuterium](https://en.wikipedia.org/wiki/Deuterium)?
It has a number of commercial and scientific uses, like nuclear reactors, spectroscopy, tracing, drugs etc

**Comment by MylesBorins** [comment]
The element system will eventually fall apart as there are not elements for
every letter.

It might be worth considering another naming scheme

**Comment by jasnell** [comment]
Sure. Any suggestions?

**Comment by bengl** [comment]
Is there actual value in still having a naming scheme? The numbers seem to be used basically everywhere.

**Comment by MylesBorins** [comment]
Animals
Comic book characters
Cities
Spices
Movie titles

**Comment by ronkorving** [comment]
- (Computer?) Scientists
- Stars and other celestial objects
- Star Wars characters (I think trademarks may be an issue though, same may be true for comic books)
- Food (Android is already exhausting confectionery names though)

**Comment by Trott** [comment]
* Muppet characters (Animal, Beaker, Camilla, Dr. Teeth...)
* Homestar Runner characters (Arrow'd Guy, Bubs, The Cheat, Drive-Thru Whale, Fhqwhgads...)

**Comment by MylesBorins** [comment]
Come on... Fhqwhgads

**Comment by klimashkin** [comment]
<details>
  <summary>Positive Words That Start With D</summary>
  <p>
DABBLE, DAIMON, DAINTIFY, DAINTY , DANCE, DANCER , DANDY, DANKE, DANKEN, DAPPER , DAPPERLY, DARING , DARINGLY, DARLING , DASHING, DASHINGLY , DAUNTLESS , DAUWTRAPPEN, DAWN, DAYEE, DAZZLE, DAZZLED, DAZZLING, DAZZLINGLY , DEAD-CHEAP, DEAD-ON, DEAN , DEAR , DEAR ME, DEARLY, DEBONAIR, DEBONAIRLY , DECENCY , DECENT, DECENTLY , DECIDING , DECISIVE , DECISIVELY, DECISIVENESS, DECK, DECLARE, DECOCT, DECORATE, DECORE, DECOROUS, DECOROUSLY , DECREE, DEDICATED, DEDUCE, DEDUCTIVE, DEED, DEEP , DEEPEN, DEEPER PART OF YOU, DEEPLY, DEFEAT, DEFEATING, DEFENCELESSNESS, DEFEND, DEFENDER, DEFER, DEFERENCE, DEFIANT, DEFIANTLY , DEFINE, DEFINITE , DEFINITELY, DEFINITIVE, DEFIX, DEFT, DEFTLY , DEIFY, DELECTABLE , DELECTABLY , DELECTATE, DELIBATE, DELIBERATE, DELIBERATELY, DELICACY, DELICATE , DELICATELY , DELICIOUS, DELICIOUSNESS, DELIGHT, DELIGHTED, DELIGHTEDLY, DELIGHTFUL , DELIGHTFULLY , DELIGHTFULNESS, DELIVER, DELUXE, DEMOCRATIC, DEMONSTRATIVE , DEMULCE, DEMULCENT, DEONERATE, DEOPPILATE, DEOSCULATE, DEPENDABILITY, DEPENDABLE , DEPENDABLY , DEPICT, DEPOSIT, DEPREICATE, DEPURATE, DEPUTE, DEPUTIZE, DERIVE, DESCRIBE, DESCRY, DESERVE, DESERVEDLY, DESERVEDNESS, DESERVING , DESERVINGLY , DESERVINGNESS, DESIGNER , DESIRABLE , DESIRABLY, DESIRE, DESIRED, DESIRING, DESIROUS , DESTINED , DESTINY, DESUME, DETACHABLE, DETACHMENT, DETECT, DETERMINANT, DETERMINATE, DETERMINATION, DETERMINE, DETERMINED , DETERMINEDLY , DEVELOP, DEVELOPED , DEVELOPING , DEVISE, DEVISEE, DEVISOR , DEVOTED, DEVOTEDLY, DEVOTEDLY , DEVOTEE, DEVOTION, DEVOTIONAL , DEVOUT , DEVOUTLY , DEW, DEXTERITY, DEXTEROUS, DEXTEROUSLY , DIADEM, DIALECTIC, DIAMOND, DIAMONDIZE, DIDACTIC , DIE-HARD, DIFFERENCE, DIFFERENT, DIFFERENTLY , DIG, DIGHT, DIGNIFIED , DIGNIFY, DIGNITY, DILIGENCE, DILIGENT , DILIGENTLY , DIMPLE, DING, DINKUM , DIP, DIPLOMATIC , DIPLOMATICALLY , DIRECT , DIRECTION, DIRECTLY , DIRECTOR, DIRT-CHEAP, DISARMING , DISARMINGLY, DISCERNING, DISCERNINGLY, DISCIPLE, DISCIPLINE, DISCIPLINED , DISCLOSE, DISCLOUD, DISCOVER, DISCOVERER, DISCOVERY, DISCREET , DISCRETE , DISCRETELY , DISCRETION, DISCRIMINATING , DISCRIMINATINGLY, DISCUMBER, DISCUSS, DISENCHANT, DISENSLAVE, DISENTANGLE, DIS-IDENTIFY, DISILLUSION, DISINVOVLE, DISNEY, DISPAND, DISPASSIONATE, DISPATCH, DISPENSE, DISPLAY, DISSOLVE, DISTINCT , DISTINCTION, DISTINCTIVE , DISTINCTIVELY , DISTINGUISHED , DISTINGUISHING, DISTRACTING, DISTRIBUTE, DISTRIBUTOR, DIVERSE , DIVERSELY , DIVERSIFIED, DIVERSITY, DIVERTED, DIVERTING , DIVINE , DIVINE LOVE, DIVINE MAGNIFICENCE, DIVINE SIGHT, DIVINE SOUL, DIVINE VOICE, DIVINE WILL, DIVINE WISDOM, DIVINELY, DIZEN, DO, DOABLE , DOCUMENT, DOER , DOES THE TRICK, DOGGONE, DOLL, DOMINANT , DOMINATE, DOMINATED, DOMINATES, DONATE, DONEE, DONOR, DOPE, DOPE CHILL OUT, DOTE, DOTE ON, DOTED ON, DOTING , DOTINGLY, DOUBTLESS , DOUGHTY , DOVELIKE, DOVETAIL, DOWN TO A FINE ART, DOWN-TO-EARTH, DOYEN, DOYENNE , DRAFT, DRAPE, DRAW, DRAWS ME BACK, DREAM, DREAMBOAT , DREAMILY, DREAMLAND, DREAMY , DRESS, DRIB, DRINK, DRIVE, DRIVEN , DRIVING , DROLL, DROOL, DROP DEAD GORGEOUS, DUB, DULCET, DULCIFY, DULCORATE, DUMBFOUNDED, DUMBFOUNDING, DUMMY-PROOF, DUPLICATE, DURABLE , DURABLY , DUTIFUL , DUTY, DYE, DYNAMIC , DYNAMICALLY, DYNAMITE, DYNAMO
  </p>
</details>
http://positivewordsresearch.com/positive-words-that-start-with-d

**Comment by riyadhalnur** [comment]
Dragonfly

**Comment by hashseed** [comment]
My first thought was also Diamond. While not an element, it is a configuration of carbon.

I also thought to skip to E, but E doesn't have any good (non-radioactive) choice either.

So a new theme does make sense. Geographic locations are probably safe wrt trademark, but ianal.

**Comment by MylesBorins** [comment]
We likely should only use ideas that we can complete an alphabet with.

Some of these are a stretch, but it is fun how many cultures are represented

## Herbs + Spices


Dill
Epazote
Fennel
Garlic
Horseradish
Italian Seasoning
Jalapeno Powder
Kaffir
Lavendar
Marjoram
Nutmeg
Oregano
Parsley
Quatre epices
Rosemary
Sage
Thyme
Urfa Biber
Vadouvan
Wasabi
Xanthan Gum
Yellow Mustard Seed
Za'atar

**Comment by Qard** [comment]
@hashseed Oh, interesting idea. Maybe we could name each release after a city from a country for which we have localization support. It'd be a great way to celebrate the diversity of our community. 😊

**Comment by MylesBorins** [comment]
### Capitol Cities

Algiers
Bangkok
Copenhagen
Dhaka
El Aioun
Funafuti
Georgetown
Hanoi
Islamabad
Jakarta
Kiev
Lima
Manila
Nassau
Ottawa
Paris
Quito
Reykjavík
Santiago
Tashkent
Ulaanbaatar
Vilnius
Washington, D.C.
😢 THERE IS NO X 😢
Yerevan
Zagreb

**Comment by hashseed** [comment]
Node.js 14 LTS "Garlic" 🤨. I could get onboard with Ginger. Also "quatre epices" just means "four spices".

I'd also like to pitch animals. But Ubuntu already has that.

Or trees. Stable.

Douglas, Eucalyptus, Fir, Gingko, Hazel, Ironwood, Joshua Tree, [...](https://www.treenames.net/common_tree_names.html)

**Comment by hashseed** [comment]
> :cry: THERE IS NO X :cry:
FWIW [Xi'an](https://en.wikipedia.org/wiki/Xi%27an) used to be a historic Chinese capital.

**Comment by benjamingr** [comment]
Capitals is too politic IMO.

Other than fun, is there any reason we need a code name to begin with?

**Comment by targos** [comment]
## Hot peppers

https://www.cayennediane.com/big-list-of-hot-peppers/

Node.js 10 "Dragon's breath"

**Comment by vdeturckheim** [comment]
Mythical anymals,
come on what could be cooler than "Node.js Unicorn" ?

**Comment by bnb** [comment]
I do think it's worth noting that we can make the decision to avoid selecting names in alphabetical order with whatever we choose next - that does seem to be the downfall of the Elements option. 😊

**Comment by bnb** [comment]
+1 to:

- Comic book characters
- Hot Peppers

**Comment by hashseed** [comment]
> I do think it's worth noting that we can make the decision to avoid selecting names in alphabetical order with whatever we choose next - that does seem to be the downfall of the Elements option.

Or we could simply arbitrarily skip letters. As long as the list is monotonically growing and the element has a stable isotope we are fine?

Dysprosium would actually work. We could also skip to Erbium or Europium. Then there is Flourine, Gold, Helium/Hydrogen...

We could also choose [mineral names](https://en.wikipedia.org/wiki/List_of_minerals) or gems.
Amethyst, Beryl, Coral, Diamond, Emerald, Flourite, Garnet, Hematite, Iolite, Jade, Kryptonite, ...?

**Comment by gabrielschulhof** [comment]
Not sure about comic book characters. Those names may be
copyright/trademark-encumbered.

https://github.com/jerryscript-project/jerryscript/pull/2013#discussion_r141041427

**Comment by addaleax** [comment]
I also like the idea of just using Deuterium since it’s pretty much “close enough” to being an element, and more importantly, its name sounds like it is one.

(Or maybe Dilithium 😉.)

**Comment by hashseed** [comment]
+1 for Dilithium.

Totally missed out on Adamantium, but we still got Kryptonite, Tiberium, Unobtanium and Vibranium.

**Comment by rvagg** [comment]
Ooo, Dilithium would be really fun, perhaps the way ahead here is to expand beyond elements and include compounds. Dilithium has added fun benefits here of course. +1 from me!

**Comment by gabrielschulhof** [comment]
+1 assuming we won't get any heat from Paramount.

**Comment by nabsource** [comment]
Hard Woods?

https://en.wikipedia.org/wiki/List_of_woods

Node.js 10 "Dogwood"
Node.js 10 "Teak"
Node.js 10 "Araucaria araucana"

**Comment by ChALkeR** [comment]
_Cryptocurrencies. There should be enough of those to choose from for every letter… :rofl:_

Seriously, «Deuterium» is pretty nice while keeping the old naming scheme.

**Comment by jkrems** [comment]
I really like the suggestion of Dilithium (shouldn't be an issue I assume since it's also a real "thing" outside of trek) and Deuterium has a nice ring to it as well.

**Comment by julianduque** [comment]
Pokemon! I would love a Node.js 10 Ditto LTS and there are plenty of those https://bulbapedia.bulbagarden.net/wiki/List_of_Pok%C3%A9mon_by_name

**Comment by ronkorving** [comment]
@julianduque Like with my proposed "Star Wars characters" and the proposed "comic book characters", "Pokemon characters" too are probably all trademarked. Probably best avoided.

**Comment by chorrell** [comment]
If you're looking for a new source of possible names to use, I would like to suggest this:

https://en.wikipedia.org/wiki/Women_in_computing

**Comment by hashseed** [comment]
> https://en.wikipedia.org/wiki/Women_in_computing

That's actually an awesome idea. But what order?

**Comment by chorrell** [comment]
Firstname, lastname? Ada (Lovelace) is a pretty obvious choice to start with if you want to restart and start with 'A' names.

**Comment by sam-github** [comment]
Don't know why we have code words at all, but I like `Dubnium`! For the irony James mentioned, and its easy to spell and to say.

**Comment by styfle** [comment]
I vote `Dubnium` because it's the easiest to say of the 3 elements and the shortest.

I think we've come to far now to pick another theme, but I would be okay with including minerals or isotopes because it feels close enough to elements.

We can wait for the rollover from `Z` to pick a brand new theme.

**Comment by mhdawson** [comment]
+1 for Dilithium.

**Comment by HRK44** [comment]
Dubnium is cool.

I'd suggest Computer Science world known name as a list, for D I'd suggest Dijkstra - shortest path between **_nodes_** in a graph :)

**Comment by Matthias-Hermsdorf** [comment]
Next time start with Hydrogen, Helium,... and correlate the version with the atomic number. You will get the radiation problem decades later. But is to late for that.

**Comment by richardlau** [comment]
> Next time start with Hydrogen, Helium,... and correlate the version with the atomic number. You will get the radiation problem decades later. But is to late for that.

I'm sure this was discussed when we originally named the first LTS release (argon) and that alphabetical order was chosen over atomic number. I can't find the issue where the decision was made, but https://github.com/nodejs/Release/issues/26 is the initial discussion and https://github.com/nodejs/node/pull/3212 landed the change.

**Comment by Announcement** [comment]
node version 10 code-name "dynamite"

**Comment by olstenlarck** [comment]
Dynamite and Diamond are brilliant. I'm more for Diamond, because it represents fully that Nodejs now is absolutely amazing and with 99% support of ES2015, and probably to the end of v10 releases - 100%.

What about Douglas? On @douglascrockford.

**Comment by ljharb** [comment]
Why were element names used in the first place when there weren't any good options for D?

If the theme is changed (which would be unfortunate imo), could at least the next few decades' worth of names be pre-selected (or at least each one narrowed down to a list of acceptable choices)?

**Comment by apapirovski** [comment]
I'm +1 on Dilithium.

But as @ljharb said, whether we're sticking with the same theme or changing it, we definitely need to assess the viability of E, F, G, etc. so we're not having this conversation again next time. 😄

**Comment by cxreg** [comment]
Dilithium is a nightmare to type btw, for any of us who will need to do so regularly

**Comment by Fishrock123** [comment]
`Dubnium` sounded fine to me, fairly simple, etc.

**Comment by ljharb** [comment]
@cxreg just as an aside, what use case necessitates typing the LTS release name regularly?

**Comment by cxreg** [comment]
@ljharb I may be an anomoly here but working at NodeSource makes it a common thing

**Comment by bnb** [comment]
I imagine that qualification makes it more common than almost any other - I write each multiple times a day 😅

**Comment by WaleedAshraf** [comment]
"Batman" maybe?

**Comment by timotew** [comment]
*TL;DR.* 
*Dubnium* is cool, But naming after animals base on major improvements might no be a bad idea.
*it doesn't always have to be in English.

I would have suggested naming an LTS animals. There is hardly an LTS focus an animal can't represent. Speed,  security,  Scalability e.t.c. I don't get to see "Carbon" around very much, It not in the download page or the CLI. I hope to be useful in these aspects.

**Comment by iSkore** [comment]
Talk about an LTS animal. Tortoise. Those things never die and it’s already the fan made Node.js mascot 😂 https://raw.githubusercontent.com/indexzero/http-server/master/screenshots/public.png

Covers the aspects @timotew mentioned minus the speed.

**Comment by olstenlarck** [comment]
Tortoise is always too controversy.

**Comment by timotew** [comment]
@olstenlarck , @iSkore  putting the controversy aside.

> It has been said that the average turtle swims at a rate of about 10-12 miles per hour but some have even been recorded to swim as fast as 22 miles per hour.

For an animal that small, that's fast. Also when people hear the name "Tortoise", They never see it as an Aqua-Terrestrial Animal and it's only judge by it speed on land, even though more than 90% of the planet is covered by water. I understand why Node maintained it, Node is everywhere now just like Tortoise.

**Comment by AyushG3112** [comment]
If we want something which will always have multiple options for each alphabet, what about **Pokemon**

**Comment by richardlau** [comment]
> Also when people hear the name "Tortoise", They never see it as an Aqua-Terrestrial Animal 

That's because while tortoises are turtles they are land-dwellers and extremely poor swimmers.

**Comment by wesleynarita** [comment]
J: Jodium, W: Wolfram !
There could be some candidates with Q in Chinese, eg. Qing (Hydrogen)

**Comment by goloroden** [comment]
This may be slightly off-topic, but this question has been asked a few times in this thread (and at other places as well), but AFAICS never got a reasonable answer:

As @bengl wrote:

> Is there actual value in still having a naming scheme? The numbers seem to be used basically everywhere.

I personally have *never* used the codename, and I personally do not know *anybody* who did. Of course, this is just me, but I also never read a blog post, an article in a magazine, watched a webcast, or anything else where anybody was referring to the code name.

So what are they good for?

(Not to mention the problem that after 26 code names we will have to start over, which again renders them somewhat useless.)

**Comment by styfle** [comment]
And this is not as hypothetical as one may initially think – Ubuntu just restarted with 17.10…)

Ubuntu names all releases, not just LTS releases like Node. So they go through the alphabet much quicker than Node does.

> Does anyone of you actually make use of the codename in everyday usage?

For people who only put LTS versions in production, the codename is useful.

The [blog posts](https://nodejs.org/en/blog/release/v8.9.0/) reference the codename.

Users can read the codename from [process.release.lts](https://nodejs.org/dist/latest-v8.x/docs/api/process.html#process_process_release).

**Comment by goloroden** [comment]
> For people who only put LTS versions in production, the codename is useful.

Why? Why is `Argon` more useful than `6 LTS` (or was it `4 LTS`?)?

> The blog posts reference the codename.

The blog posts could also reference the version numbers.

> Users can read the codename from process.release.lts.

Users can also read the version number from `process.versions.node`.

You have argued *how* people could use it, but not *why* they should do it. What is better with a name such as `Argon` than with a version number such as `4 LTS`?

(And please don't take this personally, it's just that I don't get it.)

**Comment by styfle** [comment]
> You have argued how people could use it

@goloroden I'm actually not arguing anything other than demonstrating the usage and answering your questions. "Does anyone of you actually make use of the codename in everyday usage?" "I personally do not know anybody who did"

> ...but not why they should do it. What is better with a name such as Argon than with a version number such as 4 LTS?

You're right, I don't know **why** they should do it other than that it's convenient.

**Comment by ljharb** [comment]
https://github.com/creationix/nvm *relies* on LTS lines having a name; and any LTS nvm users likely use `--lts=argon` or `lts/argon` in their `.nvmrc`, or as an alias.

Regardless of what the names are, there must be names, or very common use cases - including potentially any open source node build running on travis-ci - will break.

**Comment by goloroden** [comment]
IMHO this, too, is no reason why anybody actually *needs* those code names or what makes them superior to just using version numbers. Actually, nvm worked before code names existed 😉

Of course, practically speaking, I don't want to break anything, too, so it's fine to stay with the code names – I just don't get why they were introduced in the first place. But I think that this is out of scope of this thread, and I don't want to consume your time with this, as all of you probably have more important things to do than to discuss principles with me 😉 

So, it's perfectly fine if we get back to finding a new name. However, of course, I would like to thank you, @ljharb and @styfle, for having taken the time to reply 😊

### A suggestion

Supposed that there *will* be code names in the future, I liked the suggestion of @chorrell a lot, although famous computer science persons in general might be an idea as well. Building on top of that: One could use an approach similar to Docker, and use an adjective with a last name, both starting with the current letter. Of course, there's an npm module for that ([docker-names](https://www.npmjs.com/package/docker-names)), which contains the list of [adjectives](https://github.com/bearjaws/docker-names/blob/master/index.js#L27-L96) and [names](https://github.com/bearjaws/docker-names/blob/master/index.js#L100-L562) used by Docker. One would only need to select a pair to have the same starting letter.

To build on this, what about `dreamy` (to have a positive adjective, not something such as `distracted`) and `Donna Dubinsky` (to have a woman's name for the beginning)? This could be something such as `dreamy-dubinsky` for Node.js 10.

What do you think?

**Comment by goloroden** [comment]
PPS: Or, to honour Ryan Dahl, it could also be `dreamy-dahl` 😉

**Comment by caub** [comment]
Dungeon
Daemon
Dobson
Dragon

**Comment by tommai78101** [comment]
Decahedron

**Comment by olstenlarck** [comment]
It should have good name, because in google mostly appears the 0.10. It's very important users to start searching with just a name, not by version.

**Comment by Trott** [comment]
> It should have good name, because in google mostly appears the 0.10. It's very important users to start searching with just a name, not by version.

FWIW, 10.0.0 won't have a name. The name doesn't apply until the release goes into LTS 6 months later. 

On the other hand, searching for "nodejs carbon" does pull up the 8.0.0 release announcement first (for me, at least) even though the first "Carbon" release was 8.9.0. (4.2.0 was the first "Argon" release and 6.9.0 was the first "Boron" release.) So the point may still be valid. 

But then yet again, searching for "nodejs 10.0" pulls up this issue first (for me, at least) and not anything about 0.10. So maybe 0.10 won't cause search problems after all. 
¯\\_(ツ)_/¯

A compelling-seeming argument against code names is that name discussions dominate the tracking issues (like this one) for even-numbered releases.

**Comment by stonecobra** [comment]
+1 for filling in with Dahl.  He isn't an element, but he was a catalyst!

**Comment by Trott** [comment]
@olstenlarck FTR, duckduckgo.com search for `nodejs 10.0` pulls up this issue before any 0.10 stuff too. (But yes, as you say, it will not be that way forever.)

Anyway, I'm not opposed to codenames as much as I'm not so sure codenames should take up a ton of time and space in issues like this one.  An inflexible/deterministic way to provide codenames would be great, just so we could avoid all the "I love Diamond"/"-1 to Diamond!" conversations. "Our methodology says the codename will be Dumbledore Dumpster. No, we won't change it. No, we won't talk about better names." 

Another possibility is to redirect all conversation around codenames to a relevant discussion board and leave it to CommComm or Release team to pick one at a specific time/date after reviewing the relevant conversation.

**Comment by ofrobots** [comment]
I would like to propose that a designated team (perhaps @nodejs/lts) pick the next 10 or so codenames upfront, so that we can have all the 🚲🏠in one place in one shot.

**Comment by ljharb** [comment]
Hopefully more than 10; or at least, let’s make sure the next 1000 are something that’s merely bikesheddable.

**Comment by vsemozhetbyt** [comment]
Is this true?

https://twitter.com/shelleypowers/status/980221505954082816

**Comment by vsemozhetbyt** [comment]
If not, this needs to be corrected, as after retwitting by Axel Rauschmayer (@rauschma) this will start spreading.

**Comment by olstenlarck** [comment]
@vsemozhetbyt yes it is true. See #318 and #317.

**Comment by jasnell** [comment]
No, it's not true in the sense that the name is not yet finalized. @Trott is making a proposal that has not yet been accepted. I'm sure we'll finalize on it relatively soonish. We do have several more weeks until the release is actually cut and 6 more months until a name is actually *needed*.

**Comment by shelleyp** [comment]
Saw this proposed and accepted. 

Geez, folks? This should be trivial. This shouldn't be an issue, at all. Alex's concern about gold is on point, but for the rest? It's just names. Not even offensive names. 

Accept them and move on already. Put a stick in the dirt and say you're done.

Especially since the choice of dubnium can also be seen as honoring the first African-American to contribute to science of element identification. And it was his birthday last week. 

https://twitter.com/curiouswavefn/status/978385311615959040

One thing you all need to stop doing is dithering about issues that are not that consequential, because what does that say about issues that are important? 

And you also need to start talking publicly about Node v10 so it's not just a figurehead milestone on a chart. Coming out with the codename is an excellent start.

**Comment by hashseed** [comment]
> April 24th

Thanks for confirming. The V8 team is [relying](https://github.com/nodejs/node/issues/19354#issuecomment-373157341) on the fact that we have some time between V8 6.7 branch point and the Node.js 10 initial release.

**Comment by jasnell** [comment]
I had been planning to cut a new test build yesterday but that got delayed. I'm going to be updating the branch this afternoon and generating the test build this afternoon.

**Comment by ctel** [comment]
> April 24th
Could you also keep the [Calendar](https://calendar.google.com/calendar/ical/eln7trd6k7n6asgg49bu2vqn4s%40group.calendar.google.com/public/basic.ics) updated?
This way @hashseed and the V8 team can rely on this.

**Comment by AyushG3112** [comment]
Question: given that `Ubuntu 18.04 Bionic Beaver` is expected to go stable on `April 26th`, are we testing the builds on the current beta released  of `Ubuntu 18.04`?

**Comment by rvagg** [comment]
@AyushG3112 I did some initial testing with it in our CI infra using an early version in a container. When it goes live and our infra providers make it available we'll be adding it to the mix and dropping 17.10. There aren't expected to be any problems with 18.04 support.

**Comment by AyushG3112** [comment]
@rvagg got it, thanks!

**Comment by Huddo121** [comment]
I know it may be a bit late to throw my hat in to the ring regarding the name, but what about Dolomite, "the tough black mineral that won't cop out when there's heat all about"?

It's also an "anhydrous **carbonate** mineral" according to wikipedia

**Comment by olstenlarck** [comment]
Yea, probably too late but welcome to awesome world of the geography (everything, everywhere around us) :D

**Comment by caub** [comment]
```
4: argon
6: boron
8: carbon
10: dubnium
12: erbium
14: fluorine
16: gallium
18: helium
20: iron
22: j... krypton
...
```
